source/code/plugins/mysql_workload_lib.rb

```diff
@@ -185,7 +185,7 @@ def get_server_stats
 
     # add MySQL Worst Case RAM Usage
     server_max_ram= Hash.new
-    value = 0 #reset
+    value = 0.0 #reset
```

Comment:
is 0.0 a ruby 2.6  thing?

Reply:
This is json improvements in Ruby 2.6
an integer will be dumped as "0" not "0.0".